package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "name": "@okta/okta-signin-widget",
   "description": "The Okta Sign-In Widget",
-  "version": "6.4.1",
+  "version": "6.4.2",
```

Comment:
Can you revert this change? widget-coordinator will handle version bump during release

Reply:
sure, thank you for let me know!